external/vcm/vcm/catalog.yaml

```diff
@@ -948,7 +948,10 @@ sources:
       consolidated: True
 
   10day_c48_PIRE_ccnorm_gfsphysics_15min_may2023:
-    description: 2D physics diagnostics variables from 10-day PIRE-like X-SHiELD simulation run with ccnorm=True (cloud condensate scaled by fraction for radiation)
+    description: >-
+      2D physics diagnostics variables from 10-day PIRE-like X-SHiELD simulation run with ccnorm=True
+      (cloud condensate scaled by fraction for radiation), decorrelation length overlap,
+      and FV3GFS-like radiation scheme cloud handling
```

Comment:
I realize we didn't do this last time, but since we have it written up, would it be worth including the URL of  the scoping document in the description?